bot/replies/replies.py

```diff
@@ -49,7 +53,7 @@
 reset_photos_confirmation_message = "This will clear ALL photos for this job. Proceed?"
 
 
-def prepare_keyboard(entries, field="jobname"):
+def prepare_keyboard(entries: List[QUERY_TYPE], field: str = "jobname") -> List[OBJECT_TYPE]:
```

Comment:
Return type here should be 
Sequence[Sequence[[str](https://docs.python.org/3/library/stdtypes.html#str) | [telegram.KeyboardButton](https://docs.python-telegram-bot.org/en/stable/telegram.keyboardbutton.html#telegram.KeyboardButton)]]

https://docs.python-telegram-bot.org/en/stable/telegram.replykeyboardmarkup.html